HISTORY.rst

```diff
@@ -5,6 +5,10 @@ History
 
 * Added the following new values to the ``/payment/processor`` validation:
   ``ebs``, ``hipay``, and ``lemon_way``.
+* Added the following new values to the ``/device`` validation:
```

Comment:
This is phrased a bit weird. Maybe we should at least say "input" values or something.